src/components/toast/toast.component.js

```diff
@@ -72,12 +72,21 @@ class Toast extends React.Component {
   toastContent() {
     if (!this.props.open) return null;
 
-    const { isCenter, variant, id, as, onDismiss, children } = this.props;
+    const {
+      isCenter,
+      variant,
+      id,
+      as,
+      onDismiss,
+      children,
+      width,
+    } = this.props;
 
     const toastProps = {
       isCenter,
       variant: variant || as,
       id,
+      customWidth: width,
```

Comment:
is there a reason to not pass it as `width`?

Reply:
Styled-components filter out custom props only, those that have same name as html attributes are passed down to the element, resulting in:
`<div width="200"/>`